ann_cnn_training/training.py

```diff
@@ -24,7 +24,7 @@
 import pandas as pd
 
 from dataloader_definition import Dataset_ANN_CNN
-from model_definition import ANN_CNN
+from model_definition import ANN_CNN, ANN_CNN10
```

Comment:
As part of the refactor we should perhaps make number of layers a parameter rather than a new class?